press/saas/doctype/product_trial_request/product_trial_request.py

```diff
@@ -95,6 +95,12 @@ def after_insert(self):
 		self.capture_posthog_event("product_trial_request_created")
 
 	def on_update(self):
+		if self.has_value_changed("site") and self.site:
+			init_telemetry()
+			ph = getattr(frappe.local, "posthog", None)
+			with suppress(Exception):
+				ph and ph.alias(previous_id=self.get_email(), distinct_id=self.site)
```

Comment:
This can sometimes be problamatic? Same email will have multiple sites.

Account request and first site is better here? Both will be unique to a person/team.